services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str | None = None,
+    last_name: str | None = None,
```

Comment:
The type hint `str | None` is used here, which is only supported in Python 3.10 and later. Ensure that your environment supports this syntax or use `Optional[str]` from `typing` for compatibility with earlier versions.